### DISCUSSION TITLE
Seeking Assistance with My Suspended GitHub Account

### DISCUSSION THREAD

**Discussion by techWorld90** [community/community#142421] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Profile

### Body

Hi GitHub Community,

I’m reaching out to seek help regarding an issue with my GitHub account. My account was recently suspended, and I’m unsure about the specific reasons or what steps I should take next. Here are the details about my situation:
Background Information

    Account Usage:
        I'm a B.Tech Computer Science and Engineering student.
        I have been actively using GitHub for hosting my personal projects, code for competitions, and collaboration with peers on academic and extracurricular activities.
        I am also preparing for various programming competitions and actively working on my projects to showcase my skills.
Current Situation

My GitHub account was suspended recently, but I did not receive clear communication about the specific reasons for the suspension. I have always tried to adhere to GitHub's guidelines and community standards. I’m concerned because:

    I rely on GitHub as a key platform to host and maintain my projects.
    The suspension affects my ability to contribute to open-source projects and showcase my portfolio.
    I am eager to understand what went wrong and how I can resolve the situation.

Actions Taken

    I’ve checked my email for any notification or detailed explanation from GitHub regarding the suspension, but I couldn’t find a clear reason.
    I’ve reviewed GitHub’s Terms of Service and Community Guidelines to ensure I wasn’t violating any rules, but I’m still uncertain about what caused the suspension.
    I have not been able to access my repositories, which is crucial for my ongoing projects and preparations.

Request for Assistance

I would greatly appreciate guidance or advice from the community regarding the following:

    What steps should I take to understand the reason for the suspension?
    How can I appeal or request a review of my account suspension?
    Has anyone else faced a similar issue, and how was it resolved?

Any support or direction would be highly valuable. Thank you for taking the time to read my post, and I look forward to any assistance you can provide.

Best Regards,
Akash

**Comment by queenofcorgis** [comment, marked as answer]
Hi @techWorld90, 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place on submitted. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

Please note: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
